content/src/service/Service.ts

```diff
@@ -27,20 +27,20 @@ import { FailedDeployment, FailureReason } from './errors/FailedDeploymentsManag
 export interface MetaverseContentService {
   start(): Promise<void>
   deployEntity(
-    files: ContentFile[],
+    files: DeploymentFiles,
```

Comment:
We are now allowing to send the already hashed files, so that we avoid hashing them both at the denylist and service layer